### PR TITLE
Add test for arci-ros MoveBase

### PR DESCRIPTION
**Pull request by chatblanc-ciel**
no test yet

### CODE REVIEW

**Comment by chatblanc-ciel** [comment]
Need to change
- [ ] running roscore in test function
- [ ] rename topic name (`/cmd_vel` is used by real)
- [ ] move codes to `tests` folder

File: arci-ros/tests/test_cmdvel_base.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#[cfg(target_os = "linux")]
```
Comment (OTL) [inline arci-ros/tests/test_cmdvel_base.rs:1]: Use `#![cfg(target_os = "linux")]` instead

File: arci-ros/src/cmd_vel_move_base.rs [hunk 2]
```diff
@@ -39,3 +39,33 @@ impl MoveBase for RosCmdVelMoveBase {
...
+    use super::*;
+
+    #[tokio::test]
+    async fn test_cmd_vel() {
+        use arci::{BaseVelocity, MoveBase};
+
+        let topic_name = String::from("/cmd_vel");
+        crate::init("arci_ros_cmd_vel_test");
+
+        let sub = rosrust::subscribe(&topic_name, 1, |v: msg::std_msgs::Header| {
```
[the remark this answers is not in the document]
Comment (chatblanc-ciel) [reply arci-ros/src/cmd_vel_move_base.rs]: I'm sorry that this code will be been deleted.
I forget it!

File: arci-ros/tests/test_cmdvel_base.rs (hunk 1 above)
Comment (chatblanc-ciel) [reply arci-ros/tests/test_cmdvel_base.rs:1]: Ok!

**Review by OTL — APPROVED** [review]

**Pull request merged**